pkg/apply/apply.go

```diff
@@ -24,6 +24,15 @@ type Object interface {
 	runtime.Object
 }
 
+func RemoveObject(objs []*unstructured.Unstructured, name string, kind string) []*unstructured.Unstructured {
```

Comment:
There's already a function in `pkg/util/k8s` called `RemoveObjectByKindGroupName` that does this.
`

Reply:
Moved this logic to `pkg/network/ovn_kubernetes.go` and use `k8s.UpdateObjByGroupKindName`.